tests/integrations/coprocessor/test_checksum.rs

```diff
@@ -54,7 +54,7 @@ fn test_checksum() {
             (range, ChecksumScanOn::Index)
         };
         let request = new_checksum_request(range.clone(), scan_on);
```

Comment:
I think `ChecksumScanOn` is also unnecessary, and I wonder could we remove  `ChecksumScanOn`  in `new_checksum_request`  and other place we using it?

Reply:
ChecksumScanOn is used in metrics.